src/api.js

```diff
@@ -809,7 +809,7 @@ API.prototype.router = function(options) {
   options = _.defaults({}, options, {
     inputLimit:           '10mb',
     allowedCORSOrigin:    '*',
-    context:              {},
```

Comment:
The `context` argument to `API()` is an array of strings (names of expected context objects).  The `context` argument to `api.router()` is an object, with context values at each property.

```javascript
const api = API({
  context: ['database', 'cache'],
});

api.router({
  context: {
    'database': new Database(),
    'cache': new Cache(),
  },
});
```

So making one (the object) default to the other (the array) doesn't make much sense.